src/QuadForm/Quad/Lattices.jl

```diff
@@ -87,12 +91,15 @@ matrix over `K` of size the length of the elements of `gens`.
 
 If `gens` is empty, `gram` must be supplied and the function returns the zero lattice
 in the quadratic space over `K` with gram matrix `gram`.
+
+If $K = \mathbb{Q}$, then the output lattice is of type `ZZLat`, seen as a lattice
+over the ring $\mathbb{Z}$.
 """
-function quadratic_lattice(K::Field, gens::Vector ; gram = nothing)
+function quadratic_lattice(K::Field, gens::Vector; gram = nothing, check::Bool = true)
   if length(gens) == 0
     @assert gram !== nothing
     pm = pseudo_matrix(matrix(K, 0, nrows(gram), []))
-    L = quadratic_lattice(K, pm, gram = gram, check = false)
+    L = quadratic_lattice(K, pm, gram = gram, check = check)
```

Comment:
What is checked here? 
`quadratic_space` also has a check keyword, so the `check=check` should be passed through to the other constructors as well.

Reply:
Yes indeed now I realised that here it is not necessary, since it is checked that `pm` has full rank. But here it has rank 0 I guess we always force the check for creation of spaces, namely checking that the gram matrix is symmetric and with the correct type of entries.